plugins/flytekit-aws-batch/flytekitplugins/awsbatch/task.py

```diff
@@ -0,0 +1,71 @@
+from dataclasses import dataclass
+from typing import Any, Callable, Dict, List, Optional, Union
+
+from dataclasses_json import dataclass_json
+from google.protobuf import json_format
+from google.protobuf.struct_pb2 import Struct
+
+from flytekit import PythonFunctionTask
+from flytekit.extend import SerializationSettings, TaskPlugins
+
+
+@dataclass_json
+@dataclass
+class AWSBatch(object):
+    """
+    Use this to configure a job definition for a AWS batch job. Task's marked with this will automatically execute
+    natively onto AWS batch service.
+    Refer to AWS job definition template for more detail: https://docs.aws.amazon.com/batch/latest/userguide/job-definition-template.html
```

Comment:
also link to here - https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/batch.html#Batch.Client.register_job_definition